cryodrgn/commands/analyze_landscape_full.py

```diff
@@ -212,15 +213,15 @@ def generate_and_map_volumes(
             log(i)
         if args.downsample:
             extent = lattice.extent * (args.downsample / (D - 1))
-            vol = model.decoder.eval_volume(
+            vol = model.decoder.eval_volume(  # type: ignore  # PYR00
```

Comment:
I left the comment `PYR00` here to indicate (to myself) that this issue is identical to all the other instances of `type: ignore PYR00` that I've put in elsewhere.

I don't know what the exact problem is (yet), or its solution, but this will allow me to address this issue independently in one shot, to avoid holding back other issues/PRs that will (and have) likely accumulate. Is this acceptable? If so, I'll open up an issue that says 'remove all PYR000 type-ignores' and assign myself to it. If not, then I'll work on fixing this (and put other things on hold in the meantime).

Reply:
All instances of the `PYR00` comment for type ignores have been fixed.